tests/utils/utils.go

```diff
@@ -0,0 +1,190 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package utils
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"math/big"
+
+	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
+	"github.com/ava-labs/subnet-evm/core/types"
+	erc20destination "github.com/ava-labs/teleporter-token-bridge/abi-bindings/go/ERC20Destination"
+	erc20source "github.com/ava-labs/teleporter-token-bridge/abi-bindings/go/ERC20Source"
+	exampleerc20 "github.com/ava-labs/teleporter/abi-bindings/go/Mocks/ExampleERC20"
+	"github.com/ava-labs/teleporter/tests/interfaces"
+	teleporterUtils "github.com/ava-labs/teleporter/tests/utils"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+
+	. "github.com/onsi/gomega"
+)
+
+func DeployERC20Source(
+	ctx context.Context,
+	senderKey *ecdsa.PrivateKey,
+	subnet interfaces.SubnetTestInfo,
+	teleporterManager common.Address,
+	tokenSourceAddress common.Address,
+) (common.Address, *erc20source.ERC20Source) {
+	opts, err := bind.NewKeyedTransactorWithChainID(
+		senderKey,
+		subnet.EVMChainID,
+	)
+	Expect(err).Should(BeNil())
+	address, tx, erc20Source, err := erc20source.DeployERC20Source(
+		opts,
+		subnet.RPCClient,
+		subnet.TeleporterRegistryAddress,
+		teleporterManager,
+		tokenSourceAddress,
+	)
+	Expect(err).Should(BeNil())
+
+	teleporterUtils.WaitForTransactionSuccess(ctx, subnet, tx.Hash())
+
+	return address, erc20Source
+}
+
+func DeployERC20Destination(
+	ctx context.Context,
+	senderKey *ecdsa.PrivateKey,
+	subnet interfaces.SubnetTestInfo,
+	teleporterManager common.Address,
+	sourceSubnet interfaces.SubnetTestInfo,
```

Comment:
For helpers, I think we should require the minimum amount of information necessary. Here we just need the blockchain ID of the source subnet, not the RPC and everything else in the subnet info